etabotsite/etabotapp/urls.py

```diff
@@ -6,14 +6,21 @@
 from .views import index
 from .views import activate
 from .views import email_verification
+from .views import jira_callback
 from django.contrib.auth import views as auth_views
+from django.contrib.staticfiles.urls import staticfiles_urlpatterns
+import logging
+
 
 router = DefaultRouter()
 router.register(r'users', UserViewSet, base_name='users')
 router.register(r'projects', ProjectViewSet, base_name='projects')
 router.register(r'tms', TMSViewSet, base_name='tms')
 
-urlpatterns = [
+urlpatterns = staticfiles_urlpatterns()
```

Comment:
Would the rearranging having any effect on production deployment?

Reply:
generally yes, the order of the urlpatterns matter.
I needed to add this for proper serving of static files.